src/App.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {render, fireEvent} from '@testing-library/react';
+
+import App from './App';
+
+describe('UWB ACM Website', () => {
```

Comment:
I'd like to see this moved to a `/tests` directory, along with `Routes.test.jsx`, if that sounds good to you. I assume the `import` statements respect relative paths, including parent paths.